app/templates/client/app/index.html

```diff
@@ -94,5 +92,17 @@
         <script src="scripts/services/account.js"></script>
         <script src="scripts/controllers/homepage.js"></script>
         <!-- endbuild -->
+
+    <script>
+      // Use showJsError bower component if exists.
+      if (typeof showJSError !== 'undefined') {
+        showJSError.init({
+          title: 'JavaScript error',
+          userAgent: navigator.userAgent,
+          sendText: 'Send',
+          sendUrl: 'https://github.com/skeleton/skeleton/issues/new?title={title}&body={body}'
```

Comment:
the github URL should be replaced upon `yo hedley` -- see [other places](https://github.com/Gizra/generator-hedley/blob/master/app/index.js#L242) where is happens